core/src/main/java/com/novoda/noplayer/player/PlayerFactory.java

```diff
@@ -58,10 +67,10 @@ public Player create(DrmType drmType, DrmHandler drmHandler, boolean downgradeSe
     private Player createPlayerForType(PlayerType playerType, DrmType drmType, DrmHandler drmHandler, boolean downgradeSecureDecoder) {
         switch (playerType) {
             case MEDIA_PLAYER:
-                return mediaPlayerCreator.createMediaPlayer(context);
+                return noPlayerMediaPlayerCreator.createMediaPlayer(context);
```

Comment:
I mentioned this the no-player room but I can say again 👻 

It comes down to which layer should provide a fully instantiated player, should it be the concrete player packages (media/exo) which allows all the classes to remain package local or the noplayer root and then expose implementation details about the separate players